Pal/src/host/FreeBSD/db_sockets.c

```diff
@@ -279,54 +279,56 @@ PAL_HANDLE socket_create_handle (int type, int fd, int options,
     return hdl;
 }
 
```

Comment:
ANY address (0.0.0.0:0) was not accepted in the PAL before. But now we are allowing it to enable some use cases (especially for LTP).